### DISCUSSION TITLE
how to support TTC font file?

### DISCUSSION THREAD

**Discussion by CNV1018** [opentypejs/opentype.js#668] [Q&A]
I'm currently using this component in my project to allow user to view & select each character from the font package. It works great on TTF and OTF fonts. I understand that it doesnt support TTC font file. Can I check if there is possible to support TTC font type? Or is there any other method to have TTC fonts rendered?

**Comment by Connum** [comment]
TTC is not currently supported

**Reply by CNV1018** [reply]
is there any similar component that can support TTC? Will this TTC file be supported in anytime soon?
